create-quasar/utils/index.js

```diff
@@ -218,6 +220,36 @@ module.exports.ensureOutsideProject = function () {
   }
 }
 
+async function getShellResponse(command) {
+  try {
+    await execComm(command)
+    return true
+  } catch (e) { 
+    return false
+  }
+}
+
+module.exports.checkServeScript = async function (filePath) {
+  const linuxResponse = await getShellResponse('which quasar')
+  const windowsResponse = await getShellResponse('where quasar')
+  if (!(linuxResponse || windowsResponse)) {
+    try {          
+      const fileData = existsSync(filePath)
+      ? 
+      (readJsonSync(filePath))
+      : {};
+      logger.log("I'm here *****************")
```

Comment:
This whole approach, of adding the script by default and removing it later on, isn't good enough
But let that aside, never leave dev logging into the final code